README.md

```diff
@@ -1,2 +1,32 @@
 # php-cs
-PHP Coding style configuration for reBuy projects
+
+The file [.php_cs.dist](.php_cs.dist) contains all rules which apply to the coding standards for rebuy. You can find the description of these rules in the [PHP-CS-Fixer](https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/master/README.rst) repository. 
+
+## Examples
+
+### Travis code style check
+
+```bash
+vendor/bin/php-cs-fixer fix --config=vendor/rebuy/php-cs/.php_cs.dist -v --dry-run --using-cache=no --path-mode=intersection `git diff --name-only --diff-filter=ACMRTUXB $TRAVIS_COMMIT_RANGE`
+```
+
+### Fixing code styles
+
+```bash
+vendor/bin/php-cs-fixer fix --config=vendor/rebuy/php-cs/.php_cs.dist yourDirectory/`
```

Comment:
I like to do 

```bash
git diff --name-only master | grep ".php$" | while read -r file; do; echo "File: $file"; bin/php-cs-fixer fix --verbose --config-file=full.php_cs $file; done
```

that fixes all changed php files from `master` to current branch.